xcp/bootloader.py

```diff
@@ -432,7 +432,7 @@ def create_label(title):
             # If this fails, it is probably a string, so leave it unchanged.
             try:
                 default = menu_order[int(default)]
-            except ValueError as KeyError:
+            except (ValueError, KeyError):
```

Comment:
IMO it's not futurize being wrong here, it's the code being buggy initially due to missing parentheses (KeyError was never catched as far as I can tell, as the following test I just did shows).

```
>>> try:
...     raise MonException, "blabla lalili"
... except IOError, MonException:
...     print "Catched it"
... 
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
__main__.MonException: blabla lalili
>>>
```

Reply:
right, it warrants a pre-futurize fix instead

Reply:
Funny, now github shows my (outdated) comment here.